docs/get-started/upgrading.md

```diff
@@ -24,7 +24,7 @@ The best way of doing this is wrapping any code that invokes a client class in a
 
 ```python
 async with Session() as session:
```

Comment:
This is what I mean by "centers the Session class". Creating one is the first thing we do and it's the manager for the context of a block of code.